lib/block/operation.go

```diff
@@ -36,6 +38,16 @@ type BlockOperation struct {
 	isSaved bool
 }
 
+// Implement `encoding.BinaryMarshaler`
+func (bo BlockOperation) MarshalBinary() (data []byte, err error) {
```

Comment:
The migration should be applied before this code deployed. right?
This should be handled carefully.

Reply:
The migration *is* this code. `sebak` will refuse to start if the migration isn't applied. See the changes to `cmd/sebak/cmd/run.go`.

Reply:
I mean that the upgrade command should be applied to exiting database before the node deployed with this commit.

Did you mentioned this critical point to SRE team?

I just worried that.

Reply:
> Did you mentioned this critical point to SRE team?

I didn't see any reason to do so, as long as this PR is not merged. And even so, the error message is pretty clear.